quaternaire.csl

```diff
@@ -27,8 +27,9 @@
         <name-part name="family" text-case="uppercase"/>
         <name-part name="given" text-case="uppercase"/>
       </name>
+      <label form="short" prefix=" (" suffix=")"/>
       <substitute>
-        <text macro="editor" font-weight="bold" suffix=" (Dir.) - "/>
+        <namves variable="editor"/>
```

Comment:
and i absolutely need the suffix (Dir.) - for the editors. Deleting this, and i have (Ed.) which is impossible...

Reply:
error in namves